src/scripts/main.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve) => {
+    element.addEventListener(eventName, () => {
+      resolve(`
```

Comment:
There is a syntax error in the template literal. The backtick (`) at the beginning of the string should be placed before the string starts, not after the opening parenthesis. It should be: `resolve(`It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.`);`